tbx/static_src/javascript/components/desktop-close-menus.js

```diff
@@ -26,6 +30,15 @@ class DesktopCloseMenus {
             }
         });
 
+        if (
+            this.mobileNav.contains(e.target) ||
+            this.mobileNavToggle.contains(e.target) ||
+            this.mobileNav.classList.contains('is-visible')
```

Comment:
I think we only need to check the last one is true, i.e. 

```
if (this.mobileNav.classList.contains('is-visible')) {
    // don't close the menus (or allow the page to scroll) if the mobile menu is open
    close = false;
}
```

Worth a quick cross-browser test though.

Reply:
Thanks, I've updated it : )